src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -101,13 +108,29 @@ export class AssetsTable extends React.Component {
     return this.trashcanRefs[focusAsset.id];
   }
 
+  getCopyUrlButtons(studioUrl, webUrl) {
+    return (
+      <span>
+        {this.getCopyUrlButton(studioUrl, 'Copy Studio URL')}
+        {this.getCopyUrlButton(webUrl, 'Copy Web URL')}
+      </span>
+    );
+  }
+
+  getCopyUrlButton(url, label) {
+    return (<CopyButton
+      label={label}
+      onCopyLabel="Copied!"
+      textToCopy={url}
+    />);
+  }
+
   addSupplementalTableElements() {
     const newAssetsList = this.props.assetsList.map((asset, index) => {
       const currentAsset = Object.assign({}, asset);
       const deleteButton = (<Button
         className={[FontAwesomeStyles.fa, FontAwesomeStyles['fa-trash']]}
         label={''}
-        buttonType={'light'}
```

Comment:
The focus is really hard to see with this button type, so I've removed it. The default button type has a much clearer outline around it. I think, for the time being, it'll be easier to verify focus this way.